packages/vue/src/stories/getting-started/2. development guide/1. customization.stories.mdx

```diff
@@ -281,12 +281,12 @@ We can use the latest to disable buttons when they're not usable.
   :visible="5"
   @click="page => currentPage = page"
 >
-  <template #prev="{ isDisabled, go }">
-    <button @click="go()" :disabled="isDisabled ? true : false">prev</button>
-  </template>
-  <template #next="{ isDisabled, go }">
-    <button @click="go()" :disabled="isDisabled ? true : false">next</button>
```

Comment:
if we decide not to remove isDisabled then this would be a good change in these examples: `:disabled="isDisabled `